data/quarg/quarg outfits.txt

```diff
@@ -16,10 +16,10 @@ outfit "Nanotech Battery"
 	category "Power"
 	cost 3000000
 	thumbnail "outfit/quarg nanotech battery"
-	"mass" 50
-	"outfit space" -50
-	"energy capacity" 60000
-	description "Human engineers who first came into contact with the Quarg were astonished by the sheer efficiency of their technology. The energy density of Quarg batteries is unparalleled, storing nearly a third more energy than the best human batteries while being less than a third the size."
+	"mass" 30
+	"outfit space" -30
+	"energy capacity" 40000
+	description "Human engineers who first came into contact with the Quarg were astonished by the sheer efficiency of their technology. The energy density of Quarg batteries is unparalleled, storing almost as much energy as the best human batteries while being less than a third the size."
```

Comment:
```suggestion
	description "Human engineers who first came into contact with the Quarg were astonished by the sheer efficiency of their technology. The energy density of Quarg batteries is unparalleled, storing almost as much energy as the best human batteries while being less than half the size."
```

The largest human battery is 80t

The current stats also still only have it as marginally better than the Large Battery Module or the Crystal Capacitor. I feel it should be better than that.
![image](https://user-images.githubusercontent.com/15916854/200364156-5f274fc4-f043-4e7d-b8d5-656a95c8e256.png)

Reply:
Make it 25 size then?